compiler/luci/pass/src/ReplaceNonConstFCWithBatchMatMulPass.cpp

```diff
@@ -165,9 +168,6 @@ bool replace_fc_with_matmul(luci::CircleFullyConnected *fc)
     x = loco::must_cast<luci::CircleNode *>(fc->input());
   }
 
-  if (x->dtype() != loco::DataType::FLOAT32 || y->dtype() != loco::DataType::FLOAT32)
-    return false;
```

Comment:
This PR removes this restriction.

Now x and y can be quantized inputs. To support the case, we copy the qparam of the original FC into newly created nodes (BMM, Reshape).